rootfs/api/models.py

```diff
@@ -152,7 +142,7 @@ class App(UuidAuditedModel):
     """
 
     owner = models.ForeignKey(settings.AUTH_USER_MODEL)
-    id = models.SlugField(max_length=24, unique=True, default=select_app_name,
```

Comment:
The `default` attribute has been removed? That would remove a lot of the boilerplate in save().